### DISCUSSION TITLE
Table with most recent related column

### DISCUSSION THREAD

**Discussion by emmexx** [filamentphp/filament#9309] [Help] [answered]
I'd like to know if there's a filament way of setting up a table for 2 related models of type customer - purchases, that is a customer has many purchases, but I want to display only a field value of the latest purchase, something like:

```
customer name | customer email | most recent purchase
john doe | jd@example.com | 2023-07-12
Mary Smith | ms@email.com | 2022-12-23
```

If I use TextColumn::make('purchases.date') Filament will display the dates of all related purchases in any cell and I don't want that.

Is that something to be accomplished in Filament or should I do that in the models?

**Comment by sinnbeck** [comment, marked as answer]
Add a new relationship to just point to one specific record

https://laravel.com/docs/10.x/eloquent-relationships#has-one-of-many

**Reply by emmexx** [reply]
Thank you, I forgot about latestOfMany() and oldestofMany() in has-one-of-many relationships
